Sources/Basics/Concurrency/ConcurrencyHelpers.swift

```diff
@@ -76,3 +77,87 @@ extension DispatchQueue {
         }
     }
 }
+
+/// A queue for running async operations with a limit on the number of concurrent tasks.
+public final class AsyncOperationQueue: @unchecked Sendable {
+
+    // This implementation is adapted from the one in swift-build,
```

Comment:
Instead of adding this comment, can you please also open a PR to update the copy in SwiftBuild?